socket_wrapper.cpp

```diff
@@ -30,43 +30,60 @@ void SocketWrapper::InitSocketAddress(int family, int address, int port)
     m_socket_address.sin_port = port;
 }
 
-void SocketWrapper::SetInAddressWithStr(const char* ip_address_str)
+void Socket::SetInAddressWithStr(const char* ip_address_str)
```

Comment:
As we already have this cool function, we can create one more version of `Bind` function, which will take `address` parameter as string. I think it will be useful.